data/fields/cycleway.json

```diff
@@ -43,6 +43,10 @@
             "opposite": {
                 "title": "Contraflow bike lane",
                 "description": "A bike lane that travels in both directions on a one-way street"
+            },
+            "separate": {
+                "title": "Separate Way for cycleway",
```

Comment:
How about something like „mapped separately“ to underline the fact that is has to be present / visible / mapped at this time.

Reply:
Thanks, fits better and I have adjusted it.